docs/plugins/custom-payload-plugin.md

```diff
@@ -8,13 +8,17 @@ permalink: /plugins/custom-payload-plugin
 repo_link: /custom-payload-plugin
 ---
 
+
+
 # Custom Payload Plugin
 
-The plugin adds support for sending a custom payload format from server, to reduce the size of payload data.
-An example of such payload can be a JSON with page title and swup containers only.
+This is a plugin for [swup](https://swup.js.org/) - complete, flexible, extensible and easy to use page transition library for your web.
+
+The plugin adds support for sending a custom payload format from server, to reduce the size of payload data. 
+An example of such payload can be a JSON with page title and swup containers only. 
 This custom payload can be sent from server based on the `X-Requested-With` header which swup sets to value `swup`.
 
-## Installation
+## Instalation
```

Comment:
Same for the typos 🤠

Reply:
But Instalation *is* the typo? 😅

Reply:
But now we're adding it back in? I was trying to save time by not creating PRs in the repos that have that typo.

Reply:
Aaah! OK now I got you. Right makes sense, both comments.